bridge/slack/handlers.go

```diff
@@ -259,10 +268,6 @@ func (b *Bslack) handleTypingEvent(ev *slack.UserTypingEvent) (*config.Message,
 
 // handleDownloadFile handles file download
 func (b *Bslack) handleDownloadFile(rmsg *config.Message, file *slack.File) error {
-	if b.fileIsAvailable(file) {
```

Comment:
We can't remove this check. If we get here there might be some files that are cached and others are not (if you follow my comment on your addition to the `skipMessageEvent()` method). So we need to filter out (again) the ones that are available in the cache.

Reply:
@patcon this comment still needs addressing. 🙂 

My suggestion would be to keep it as is but modifying the call to correspond to `b.fileWasRecentlyCached()`.